tests/transforms/test_decompositions.py

```diff
@@ -55,53 +62,86 @@ def test_zyz_decomposition(self, U, expected_gate, expected_params):
         obtained_gates = zyz_decomposition(U, Wires("a"))
 
         assert len(obtained_gates) == 1
-
         assert isinstance(obtained_gates[0], expected_gate)
         assert obtained_gates[0].wires == Wires("a")
         assert qml.math.allclose(obtained_gates[0].parameters, expected_params)
 
+        if obtained_gates[0].num_params == 1:
```

Comment:
I've added matrix equivalence checks to all the tests now... these should have been there from the start :facepalm: